src/ansi.rs

```diff
@@ -97,6 +109,15 @@ pub fn parse_key_press(data: &[u8]) -> Option<(KeyPress, usize)> {
         }
     }
 
+    // Ping response is \x1b[12;34R when cursor is at line 12 column 34
+    if data.starts_with(b"\x1b[") {
+        if let Some(i) = data.iter().position(|c| *c == b'R') {
```

Comment:
Maybe this should also be limited to first 10 bytes. Could be slow, if someone sends huge amounts of numbers and semicolons.